tests/openbb_terminal/core/log/collection/test_log_sender.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from openbb_terminal.core.log.collection import log_sender
+from openbb_terminal.core.log.generation.settings import (
+    Settings,
+    AppSettings,
+    AWSSettings,
+    LogSettings,
+)
+
+settings = Settings(
+    app_settings=AppSettings(
+        commit_hash="MOCK_COMMIT_HASH",
+        name="MOCK_COMMIT_HASH",
+        identifier="MOCK_COMMIT_HASH",
```

Comment:
Not mocking `commit_hash`

Reply:
I copied this from what you sent me. Let me know how I should change!